02-accessing_files_in_s3.Rmd

```diff
@@ -26,7 +26,41 @@ We have also developed an R Package called `s3tools` that makes it easier to int
 See the [documentation](https://github.com/moj-analytical-services/s3tools) for
 further details.
 
-## Uploading data to S3
+## Importing data from S3 into JupyterLab
+
+Pandas can import straight from S3:
+
+    pd.read_csv("s3://alpha-everyone/iris.csv")
+
+This requires you to install a couple of libraries:
+
+    pip install pandas s3fs
```

Comment:
Users won't actually need  to install `pandas` manually because it's already installed in both the datascience notebook and the all spark notebook.